ui/apps/platform/src/Containers/Vulnerabilities/types.ts

```diff
@@ -81,3 +81,18 @@ export type ObservedCveMode = (typeof observedCveModeValues)[number];
 export function isObservedCveMode(value: unknown): value is ObservedCveMode {
     return observedCveModeValues.some((mode) => mode === value);
 }
+
+export type VerifiedStatus =
+    | 'CORRUPTED_SIGNATURE'
+    | 'FAILED_VERIFICATION'
+    | 'GENERIC_ERROR'
+    | 'INVALID_SIGNATURE_ALGO'
+    | 'UNSET'
+    | 'VERIFIED';
+
+export type SignatureVerificationResult = {
+    status: VerifiedStatus;
+    verificationTime: string;
+    verifiedImageReferences: string[];
+    verifierId: string;
```

Comment:
For future readers, what do you think about comment at the end of the line that this is signature integration